common/p_setup.cpp

```diff
@@ -1659,6 +1664,13 @@ void P_SetupLevel (char *lumpname, int position)
 		wminfo.maxfrags = 0;
 	wminfo.partime = 180;
 
+	// reset completionist counts to -1 which is translated to level.total_monsters and level.total_secrets in p_lnspec.cpp:
+	sv_coop_completionist_kills.ForceSet(-1.0f);
+	sv_coop_completionist_secrets.ForceSet(-1.0f);
```

Comment:
These `-1` sentinel values are convenient for the server to know to look up the value for the current map but for rcon users reading a `-1` values tells them nothing useful and is confusing. It'd be better to reset the cvar counts on map load to their actual values and avoid `-1`.